optaplanner-quickstarts/quarkus-school-timetabling/build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id "io.quarkus" version "1.7.0.Final"
+    id "io.quarkus" version "1.8.2.Final"
```

Comment:
[There](https://github.com/kiegroup/optaplanner/blob/2c1e0c1e729ba26df4242fdfd723563ba56805b8/optaplanner-quickstarts/quarkus-school-timetabling/build.gradle#L17) is also the universe-bom version, but I guess it's not yet available, is it?

Reply:
for now, I am leaving using the universe bom for our quickstarts of the complexity :)
We can switch to it when we figure out a good way to be able to use it reliably...